src/middleware/incontact/src/incontact.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * (C) Copyright IBM Corp. 2020.
+ *
+ * Licensed under the MIT License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ */
+
+import R from 'ramda';
+import request from 'request-promise';
```

Comment:
`request-promise` has been deprecated because it extends the now deprecated request package some of the suggested alternatives are https://www.npmjs.com/package/request-promise

Reply:
replaced